src/database/FirehoseConnector.js

```diff
@@ -187,6 +190,7 @@ class FirehoseConnector {
             meetingUniqueId,
             endpointId,
             sessionDuration,
+            conferenceDuration,
```

Comment:
update the redshift-schema.sql file with the new field, also add it to our "Notes for next deploy" document

Reply:
Fair point. I'll make these changes.